frontend/server/src/Controllers/Problem.php

```diff
@@ -4742,6 +4743,29 @@ public static function getProblemListForSmarty(
         ];
     }
 
+    /**
+     *
+     * @return array{smartyProperties: array{payload: ProblemListCollection, title: \OmegaUp\TranslationString}, entrypoint: string}
+     *
+     * @omegaup-request-param string $problem_level_tag
+     */
```

Comment:
Nit para que no se queje psalm:
```suggestion
    /**
     * @return array{smartyProperties: array{payload: ProblemListCollection, title: \OmegaUp\TranslationString}, entrypoint: string}
     */
```